Core/include/PPM.h

```diff
@@ -1,21 +1,18 @@
 #pragma once
 #include <fstream>
 #include <iostream>
-#include <cmath>
+#include "Vec3/Vec3.h"
 
 class PPM
```

Comment:
This should probably be a POD, unless you really like OOP classes.

Reply:
I do but does it affect performance ?

Reply:
Not if done well. 🤓